__init__.py

```diff
@@ -44,7 +44,8 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
-    os.path.join(HOME_DIR, 'scripts/'),
+    'scripts',
+    os.path.join(HOME_DIR, 'scripts/')
 ]
```

Comment:
Okey, I get what you mean. In this case we assume that only CLI is the only way to create a script.
This sounds good but some geeky individuals want to manually create their own `.json` scripts in the current `"./"` directory and then run them. 

That being said, to account for such rare cases, I have thought of introducing a keyword `"json_type": "retriever_script" `. This can enable us to only compile scripts made for the retriever. 
I have sent you the PR but it is failing on travis and appevyor but passing tests locally.